chanreg.md

```diff
@@ -1,228 +1,191 @@
+  highlight that member's involvement or association with their group. For
+  example, Libera.Chat staff each have cloaks beginning with `libera/staff/`
+  and official bots managed by Libera.Chat have cloaks beginning with
+  `libera/bot/` to represent their respective associations with Libera.Chat.
+  Formal community registration cloaks will begin with an `about/` prefix to
+  indicate that the community is not official. Group contacts can audit the
+  list of users with cloaks under their project at any time with
+  `/msg NickServ LISTGROUPCLOAKS`.
+- We offer to send messages about important releases or events of formally
+  registered project as a "wallop" message to our users.
+
+### Group contacts
+
+Group contacts are the people ultimately responsible for your project here on
+Libera.Chat. They are empowered to claim channels within your namespace,
+assign cloaks to other users, and act as liaisons between Libera.Chat staff
+and your project. We **strongly recommend** having between 2-4 group contacts
+and that at least one group contact is publicly listed to act as a point of
+contact for any users interested in your project.
+
+Group contacts must have registered accounts with NickServ on Libera.Chat.
+Usually, you will want to establish yourself as one of the group contacts if
+you are formally registering a group with us. The list of group contacts for
+your project or community can be modified later by sending an email to
+<projects@libera.chat> as the need arises.
+
+Group contacts can be either publicly listed or hidden, and this choice can be
+made individually per group contact. Publicly listed contacts will be shown in
+`/msg ChanServ INFO` output for all channels within your group's channel
+namespaces. Hidden group contacts are known only to you and Libera.Chat staff.
+
+### Registering a group
+
+The [Projects & Community team](#the-projects--community-team) will act as
+your liaisons during the formal group registration process and throughout your
+tenure on the Libera.Chat network. If you wish to discuss the registration
+process, find out if Libera.Chat could be a good fit for your group or
+register your group, please feel free to drop any of the team members a line
+on IRC. If none are online, feel free to reach out to staff in general and
+they will forward your request to the team, which will get back to you as soon
+as possible. Once you have discussed your project or community with a staff
+member, you’ll be asked to email <projects@libera.chat> with the details of
+your request. Please do not email us without having first discussed your
+registration on IRC with a member of staff.
+
+Please note that we only register established projects and communities. If
+your project or community does not yet exist, you are still free to register a
+channel with ChanServ to discuss and bootstrap your project or community. Once
+you feel you've gotten established enough to move forward with a formal
```

Comment:
I'm finding "gotten" slightly jarring as I seem to anticipate "become". It's been used in another of our docs and i found it similarly noticeable there.

I assume it's a regional thing, but am unsure whether one is more understandable than the other for non-native English speakers.